python/ray/tune/trial_runner.py

```diff
@@ -380,7 +381,8 @@ def add_trial(self, trial):
     def debug_string(self, delim="\n"):
         messages = [
             self._scheduler_alg.debug_string(),
-            self.trial_executor.debug_string()
+            self.trial_executor.debug_string(),
+            trial_progress_str(self.get_trials()),
```

Comment:
This was broken in some unit tests I was debugging.

Reply:
huh, can you elaborate by whats broken? this results in double printing the trial table since `trial_progress_str` is called in `progress_reporter.py`

Reply:
@ericl oh i see you want it included in the debug str, i guess that's reasonable. i'll push a hotfix to workaround that